sdk/dynatrace/Azure.ResourceManager.Dynatrace/src/Generated/DynatraceSingleSignOnResourceData.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+using Azure.ResourceManager.Dynatrace.Models;
+using Azure.ResourceManager.Models;
+
+namespace Azure.ResourceManager.Dynatrace
+{
+    /// <summary> A class representing the DynatraceSingleSignOnResource data model. </summary>
+    public partial class DynatraceSingleSignOnResourceData : ResourceData
+    {
+        /// <summary> Initializes a new instance of DynatraceSingleSignOnResourceData. </summary>
+        public DynatraceSingleSignOnResourceData()
+        {
+            AadDomains = new ChangeTrackingList<string>();
+        }
+
+        /// <summary> Initializes a new instance of DynatraceSingleSignOnResourceData. </summary>
+        /// <param name="id"> The id. </param>
+        /// <param name="name"> The name. </param>
+        /// <param name="resourceType"> The resourceType. </param>
+        /// <param name="systemData"> The systemData. </param>
+        /// <param name="singleSignOnState"> State of Single Sign On. </param>
+        /// <param name="enterpriseAppId"> Version of the Dynatrace agent installed on the VM. </param>
+        /// <param name="singleSignOnUri"> The login URL specific to this Dynatrace Environment. </param>
+        /// <param name="aadDomains"> array of Aad(azure active directory) domains. </param>
+        /// <param name="provisioningState"> Provisioning state of the resource. </param>
+        internal DynatraceSingleSignOnResourceData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, SingleSignOnStates? singleSignOnState, string enterpriseAppId, Uri singleSignOnUri, IList<string> aadDomains, ProvisioningState? provisioningState) : base(id, name, resourceType, systemData)
+        {
+            SingleSignOnState = singleSignOnState;
+            EnterpriseAppId = enterpriseAppId;
+            SingleSignOnUri = singleSignOnUri;
+            AadDomains = aadDomains;
+            ProvisioningState = provisioningState;
+        }
+
+        /// <summary> State of Single Sign On. </summary>
+        public SingleSignOnStates? SingleSignOnState { get; set; }
+        /// <summary> Version of the Dynatrace agent installed on the VM. </summary>
+        public string EnterpriseAppId { get; set; }
```

Comment:
Same question about the type.

Reply:
And by the way, the description of this property in the swagger seems to be wrong.

Reply:
Guid
Yes, the description is wrong. I'll note it and update it in the swagger in the next version.
Correct Description: Application Id of the enterprise app used for SSO